cmd/metacache-server-pool.go

```diff
@@ -324,7 +324,7 @@ func (z *erasureServerPools) listMerged(ctx context.Context, o listPathOptions,
 		return len(oMeta.versions) > len(eMeta.versions)
 	})
 
-	cancelList()
+	//cancelList()
```

Comment:
Again, this is needed if an error occurs in mergeEntryChannels. For now it is AFAICT only upstream cancelation.

This change should not affect anything.